2-extra/2-slideshow-extra/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Slideshow</title>
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Image Carousel</h1>
+    <img src="" id="carousel-img" width="300" height="250" />
+    <div>
+      <label for="second-input">Auto Time(second): </label
+      ><input type="number" id="second-input" min="1" value="1" />
```

Comment:
You should pay attendion to naming of your HTML element ID. What is the meaning of `second-input`? It will be even more confusing if you have `third-input`, `forth-input`, ... Instead, give it a meaningful name, for example `auto-time`